providers/default/ciphers/cipher_tdes_default.c

```diff
@@ -0,0 +1,29 @@
+/*
```

Comment:
I've noticed a liking for extra suffixes in file names, things like `_generic` or `_default`...  other ciphers with the "default" modes are named `cipher_{algo}.[ch]`...

Reply:
This is because there are 2 files  called cipher_tdes.

A common one and a default one.

Reply:
If you have suggestions on what to name these 2 files, let me know..

Reply:
Oooohhhhhhhhh!  I didn't notice that this was in the default provider...  Ok then, never mind!